scripts/vscode.patch

```diff
@@ -379,6 +379,50 @@ index cf9154ce7d6..5c954f75f8e 100644
  
  	public static validateOptions(options: IEditorOptions): ValidatedEditorOptions {
  		const result = new ValidatedEditorOptions();
+diff --git a/src/vs/editor/contrib/find/browser/findController.ts b/src/vs/editor/contrib/find/browser/findController.ts
+index ce1cdb9084e..0836c1d3f98 100644
+--- a/src/vs/editor/contrib/find/browser/findController.ts
++++ b/src/vs/editor/contrib/find/browser/findController.ts
+@@ -259,7 +259,7 @@ export class CommonFindController extends Disposable implements IEditorContribut
+ 		} else {
+ 			if (this._editor.hasModel()) {
+ 				const selections = this._editor.getSelections();
+-				selections.map(selection => {
++				selections.forEach(selection => {
+ 					if (selection.endColumn === 1 && selection.endLineNumber > selection.startLineNumber) {
+ 						selection = selection.setEndPosition(
+ 							selection.endLineNumber - 1,
+@@ -270,7 +270,7 @@ export class CommonFindController extends Disposable implements IEditorContribut
+ 						return selection;
+ 					}
+ 					return null;
+-				}).filter(element => !!element);
```

Comment:
I'm not sure what was attempted with that filter or if I missed something :shrug:

Reply:
I have the feeling they meant `selections = selections.map(selection => {`